src/components/date-filter/date-filter-directive.tpl.html

```diff
@@ -1,14 +1,14 @@
 <li class="dropdown" refresh-on="filterChanged" refresh-action="vm.updateFilterDisplayName()">
   <a class="dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false" refresh-on="filterChanged" refresh-action="vm.updateFilterDisplayName()">
-    {{vm.filteredDisplayName}} <span class="caret"></span>
+    {{vm.filteredDisplayName | translate}} <span class="caret"></span>
```

Comment:
Not sure if this makes sense to translate this, guess use our best judgement but it could be any computed date time range. If it falls back if the key isn't found then I guess that's fine :)

Reply:
This is default show. Not use filter is show english. So I translate this.
![image](https://user-images.githubusercontent.com/7100363/30754229-8c05aabe-9ff4-11e7-9416-92bb85b02a59.png)